### PR TITLE
Webpack migration

### PR DESCRIPTION
**Pull request by roonyh**
This fixes the built and app seems to work fine!

The worker is commented out and needed to be fixed. Working on it. More info in commit messages.

### CODE REVIEW

File: modules/web/js/ballerina/views/ballerina-file-editor.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ define(['lodash', 'jquery', 'log', './ballerina-view', './service-definition-vie
         './../visitors/source-gen/ballerina-ast-root-visitor','./../visitors/symbol-table/ballerina-ast-root-visitor', './../tool-palette/tool-palette',
         './../undo-manager/undo-manager','./backend', './../ast/ballerina-ast-deserializer', './connector-definition-view', './struct-definition-view',
         './../env/package', './../env/package-scoped-environment', './../env/environment', './constant-definitions-pane-view', './../item-provider/tool-palette-item-provider',
-        './package-definition-pane-view','./import-declaration-view', './type-mapper-definition-view', 'alerts', 'typeahead'],
+        './package-definition-pane-view','./import-declaration-view', './type-mapper-definition-view', 'alerts', 'typeahead.js'],
```
Comment (kaviththiranga) [inline modules/web/js/ballerina/views/ballerina-file-editor.js]: was this change intentional?

Comment (kaviththiranga) [inline modules/web/js/ballerina/views/ballerina-file-editor.js]: oh, ignore this comment,

File: modules/web/js/ballerina/utils/ace-worker.js [hunk 2]
```diff
@@ -16,43 +16,44 @@
...
+    //             };
+    //
+    //             request.open('POST', backends.services.validator.endpoint, true);
+    //             request.setRequestHeader("Content-type", "application/json");
+    //             request.send(JSON.stringify(content));
+    //         } else {
+    //             this.sender.emit("lint", []);
+    //         }
+    //     };
+    // }).call(WorkerModule.prototype);
```
Comment (hemikak) [inline modules/web/js/ballerina/utils/ace-worker.js]: Shall we remove this code if it is not needed ?

Comment (roonyh) [inline modules/web/js/ballerina/utils/ace-worker.js]: This has been un-commented on a follow up commit once the ace editor worker made to work with the webpack build.
